src/server/player_sao.cpp

```diff
@@ -463,36 +463,33 @@ void PlayerSAO::rightClick(ServerActiveObject *clicker)
 	m_env->getScriptIface()->on_rightclickplayer(this, clicker);
 }
 
-void PlayerSAO::setHP(s32 hp, const PlayerHPChangeReason &reason, bool send)
+void PlayerSAO::setHP(s32 orig_hp, const PlayerHPChangeReason &reason, bool from_client)
```

Comment:
"original" may mislead to assume that it's the PlayerSAO's original HP.
Why is this rename necessary? This variable becomes unused after `hp`'s initialization. If you'd like to keep the variables separated, I would suggest `new_hp` or `target_hp`.

Reply:
`target_hp` is good